roll/__init__.py

```diff
@@ -300,91 +345,201 @@ def cookies(self):
             self._cookies = self.app.Cookies()
         return self._cookies
 
+    def __bytes__(self):
```

Comment:
Tricky but elegant 😁 If I remember correctly we evaluated that option with @yohanboniface but I cannot remember why we discarded it at the end. Did you remember mate?